modules/api_docs/help/api_docs.md

```diff
@@ -0,0 +1,12 @@
+# API documentation module
```

Comment:
I was going to say this needs to be rewritten in the format of our other READMEs, but then I realized it's the help, not the README.. this module needs a README.md (in the form of our other module READMEs..)

Reply:
done